src/assets/i18n/en.json5

```diff
@@ -692,6 +692,26 @@
 
   "clarin.license.agreement.signer.ip.address": "Ip Address",
 
+  "clarin.license.agreement.signer.SEND_TOKEN": "E-mail",
+
+  "clarin.license.agreement.signer.NAME": "Name (given name and surname)",
+
+  "clarin.license.agreement.signer.DOB": "Date of birth",
+
+  "clarin.license.agreement.signer.ADDRESS": "Address",
+
+  "clarin.license.agreement.signer.COUNTRY": "Country",
+
+  "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+
+  "clarin.license.agreement.signer.ORGANIZATION": "Organization",
+
+  "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organization",
+
+  "clarin.license.agreement.signer.INTENDED_USE": "Intended use",
+
+  "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Name and seat (address) of your academic institution",
```

Comment:
seat?

Reply:
https://www.lawinsider.com/dictionary/seat-of-the-institution

Reply:
I just copied it from the UFAL: https://github.com/ufal/clarin-dspace/blob/clarin-dev/dspace/modules/xmlui/src/main/webapp/i18n/messages_cs.xml#L1972